src/main/public/stylesheets/components/_phone.scss

```diff
@@ -1,28 +1,28 @@
-.mobile-show,
-.mobile-table-status {
+.phone-show,
```

Comment:
as above, this shouldn't be changing as it relates to how things are displayed on a mobile and the other places in this file